parlai/core/teachers.py

```diff
@@ -112,6 +114,11 @@ class FixedDialogTeacher(Teacher):
 
 
     """
+    class AttrDict(dict):
+        def __init__(self, *args, **kwargs):
```

Comment:
a comment explaining this?

Reply:
move to utils where it belongs and added a comment.